examples/form.rs

```diff
@@ -0,0 +1,24 @@
+//! Forms
+//!
+//! Dioxus forms deviate slightly from html, automatically returning all named inputs
+//! in the "values" field
+
+use dioxus::prelude::*;
+
+fn main() {
+    dioxus::desktop::launch(app);
+}
+
+fn app(cx: Scope) -> Element {
+    cx.render(rsx! {
+        div {
+            h1 { "Form" }
+            form {
+                oninput: move |ev| println!("{:?}", ev.values),
```

Comment:
why we use thie oninput event? I think there have a special event: `onsubmit` for form.

Reply:
Just updated to showcase how both onsubmit and oninput support "values"